add instance_group_manager output

adrian-gierakowski opened the pull request:
needed to create a stateful MIG in conjunction with https://www.terraform.io/docs/providers/google/r/compute_region_per_instance_config.html

morgante commented on autogen/outputs.tf.tmpl:
```diff
@@ -25,3 +25,8 @@ output "instance_group" {
+
+output "instance_group_manager" {
+  description = "An instance of google_compute_region_instance_group_manager of the instance group."
+  value       = google_compute_region_instance_group_manager.mig
```
This will need to differ based on. the module name. (See line 26).

adrian-gierakowski replied:
addressed in 2752517d2de110c12e6c2bd04f02b984d0282ad0

adrian-gierakowski commented:
@morgante I've addressed your comment but one of the Cloudbuild jobs is still failing. Not sure why as I cannot see the logs. I tried to run tests locally following instructions in [CONTRIBUTING.md](/CONTRIBUTING.md) however encountered a bunch of errors while running `make docker_test_prepare` and have no time to work through them.

adrian-gierakowski commented:
anything I could to to push this through?

morgante commented:
@adrian-gierakowski We just need tests to pass, so if you can get those green again that would be good.

morgante reviewed (approved):
Looks like tests are passing now.
